<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>Publications | Shi Lab @ CNCB</title> <meta name="author" content=" Shi Lab"> <meta name="description" content="* indicates equal contributions, # indicates corresponding author.&lt;h6&gt;An up-to-date list is available on &lt;a href='https://scholar.google.com/citations?user=v04jJXoAAAAJ&amp;hl=en'&gt;Google Scholar&lt;/a&gt;.&lt;/h6&gt;"> <meta name="keywords" content="noncoding RNA, bioinformatics, embryo development"> <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%F0%9F%A7%AC&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css"> <link rel="canonical" href="https://junchaoshi.github.io//publications/"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js"></script> <script src="/assets/js/dark_mode.js"></script> </head> <body class="fixed-top-nav sticky-bottom-footer"> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/">Shi Lab @ CNCB</a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">About</a> </li> <li class="nav-item "> <a class="nav-link" href="/research">Research</a> </li> <li class="nav-item "> <a class="nav-link" href="/team/">People</a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">Publications<span class="sr-only">(current)</span></a> </li> <li class="nav-item "> <a class="nav-link" href="/repositories/">Softwares</a> </li> <li class="nav-item "> <a class="nav-link" href="/contact/">Contact</a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fas fa-moon"></i> <i class="fas fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title">Publications</h1> <p class="post-description">* indicates equal contributions, # indicates corresponding author.</p> <h6>An up-to-date list is available on <a href="https://scholar.google.com/citations?user=v04jJXoAAAAJ&amp;hl=en" rel="external nofollow noopener" target="_blank">Google Scholar</a>.</h6> </header> <article> <script type="text/javascript" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="publications"> <h6> <a href="#works_at_CNCB">🧑‍🏫Works at CNCB</a> <a href="#works_at_UCR">👨‍🎓Works at UCR</a> <a href="#works_at_UNR">🧑Works at UNR</a> <a href="#works_at_IOZ">👶Works at UCAS</a> </h6> <a name="works_at_CNCB"></a> <h1>Works at CNCB</h1> <h2 class="year">2025</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr h-100"><abbr class="badge">Sci China Life Sci</abbr></div> <div id="zhang2025scls" class="col-sm-8 h-100"> <div class="title">Peripheral blood RNA modifications as a novel diagnostic signature for polycystic ovary syndrome </div> <div class="author">Liwen Zhang*, Xinxin Liu*, Yu zhang*, Lang Qin*, Shijia Pan, Xueqi Yan, Sen Dong, Zerong Feng, Song-jia Fan, Rusong Zhao, Xueying Gao, Shigang Zhao#, <b>Junchao Shi</b>#, Han Zhao#, Ying Zhang#, and Zi-Jiang Chen</div> <div class="periodical"> <em>Science China Life Sciences</em> 2025 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="abstract hidden"> <p>Polycystic Ovary Syndrome (PCOS) is the most prevalent ovulatory and endocrine disorder affecting reproductive-aged women, yet the absence of a specific, rapid molecular diagnostic marker results in diagnostic delays and inaccuracies. Given the critical role of RNA modifications in disease pathology, this study utilized a high-throughput RNA modification profiling platform to investigate 15 types of peripheral blood RNA modification patterns in individuals with ovulatory disorders, including PCOS and Primary Ovarian Insufficiency (POI), and control subjects. Our results revealed that distinct modification profiles correspond to specific disease states, with significant shifts in RNA modification inter-correlations observed across conditions. Additionally, specific RNA modifications were associated with clinical features, such as serum levels of testosterone and the follicle number per ovary (FNPO). To optimize diagnostic precision, we evaluated various machine learning models, identifying that combining m6A and m7G modifications in a light gradient boosting machine model achieves the highest accuracy in distinguishing PCOS, outperforming traditional diagnostic markers. This highlights the potential of RNA modification profiling as a novel, high-accuracy diagnostic tool for PCOS in clinical settings.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">zhang2025scls</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Peripheral blood RNA modifications as a novel diagnostic signature for polycystic ovary syndrome }</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Zhang, Liwen and Liu, Xinxin and zhang, Yu and Qin, Lang and Pan, Shijia and Yan, Xueqi and Dong, Sen and Feng, Zerong and Fan, Song-jia and Zhao, Rusong and Gao, Xueying and Zhao, Shigang and Shi, Junchao and Zhao, Han and Zhang, Ying and Chen, Zi-Jiang}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Science China Life Sciences}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1--4}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Nature Publishing Group}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr h-100"><abbr class="badge">Nat Protoc</abbr></div> <div id="shi2025np" class="col-sm-8 h-100"> <div class="title">Optimized identification and characterization of small RNAs with PANDORA-seq</div> <div class="author"> <b>Junchao Shi</b>#, Yunfang Zhang#, Yun Li, Liwen Zhang, Xudong Zhang, Menghong Yan, <a href="http://qichen-lab.info/" rel="external nofollow noopener" target="_blank">Qi Chen</a>, and Ying Zhang#</div> <div class="periodical"> <em>Nature Protocols</em> 2025 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="abstract hidden"> <p>Small noncoding RNAs (sncRNAs) are a diverse group of RNAs including small interfering RNAs (siRNAs), microRNAs (miRNAs), PIWI-interacting RNAs (piRNAs) and RNAs derived from structured RNAs like transfer RNAs (tRNAs), ribosomal RNAs (rRNAs), and others. These sncRNAs have varied termini and RNA modifications, which can interfere with adaptor ligation and reverse transcription during cDNA library construction, hindering detection of many types of sncRNAs by standard small RNA sequencing methods. To address this limitation, PANDORA-seq introduces a refined methodology. The procedure includes sequential enzymatic treatments of size-selected RNAs with T4PNK and AlkB, which effectively circumvent the challenges presented by the ligation-blocking termini and reverse transcription-blocking RNA modifications, followed by tailored small RNA library construction protocols and deep sequencing. The obtained datasets are analyzed with the SPORTS pipeline, which can comprehensively analyze various types of sncRNAs beyond the traditionally studied classes, to include those derived from various parental RNAs (e.g., from tRNA and rRNA), as well as output the locations on the parental RNA from which these sncRNAs are derived. The entire protocol takes  7 days, depending on the sample size and sequencing turnaround time. PANDORA-seq provides a transformative tool to further our understanding of the expanding small RNA universe and to explore the uncharted functions of sncRNAs.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">shi2025np</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Optimized identification and characterization of small RNAs with PANDORA-seq}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Shi, Junchao and Zhang, Yunfang and Li, Yun and Zhang, Liwen and Zhang, Xudong and Yan, Menghong and Chen, Qi and Zhang, Ying}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature Protocols}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1--4}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Nature Publishing Group}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41596-025-01158-4}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1038/s41596-025-01158-4" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1038/s41596-025-01158-4" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li> </ol> <h2 class="year">2024</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr h-100"><abbr class="badge">Nat Cell Biol</abbr></div> <div id="kodali2024rna" class="col-sm-8 h-100"> <div class="title">RNA sequestration in P-bodies sustains myeloid leukaemia</div> <div class="author">Srikanth Kodali*, Ludovica Proietti*, Gemma Valcarcel, Anna V López-Rubio, Patrizia Pessina, Thomas Eder, <b>Junchao Shi</b>, Annie Jen, Núria Lupión-Garcia, Anne C Starner, Mason D Bartels, Yingzhi Cui, Caroline M Sands, Ainoa Planas-Riverola, Alba Martínez, Talia Velasco-Hernandez, Laureano Tomás-Daza, Bernhard Alber, Gabriele Manhart, Isabella Maria Mayer, Karoline Kollmann, Alessandro Fatica, Pablo Menendez, Evgenia Shishkova, Rachel E Rau, Biola M Javierre, Joshua Coon, <a href="http://qichen-lab.info/" rel="external nofollow noopener" target="_blank">Qi Chen</a>, Eric L Van Nostrand, Jose L Sardina, Florian Grebien#, and Bruno Di Stefano#</div> <div class="periodical"> <em>Nature Cell Biology</em> 2024 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.nature.com/articles/s41556-024-01489-6" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2024/kodali2024rna.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>Post-transcriptional mechanisms are fundamental safeguards of progenitor cell identity and are often dysregulated in cancer. Here, we identified regulators of P-bodies as crucial vulnerabilities in acute myeloid leukaemia (AML) through genome-wide CRISPR screens in normal and malignant haematopoietic progenitors. We found that leukaemia cells harbour aberrantly elevated numbers of P-bodies and show that P-body assembly is crucial for initiation and maintenance of AML. Notably, P-body loss had little effect upon homoeostatic haematopoiesis but impacted regenerative haematopoiesis. Molecular characterization of P-bodies purified from human AML cells unveiled their critical role in sequestering messenger RNAs encoding potent tumour suppressors from the translational machinery. P-body dissolution promoted translation of these mRNAs, which in turn rewired gene expression and chromatin architecture in leukaemia cells. Collectively, our findings highlight the contrasting and unique roles of RNA sequestration in P-bodies during tissue homoeostasis and oncogenesis. These insights open potential avenues for understanding myeloid leukaemia and future therapeutic interventions.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">kodali2024rna</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{RNA sequestration in P-bodies sustains myeloid leukaemia}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Kodali, Srikanth and Proietti, Ludovica and Valcarcel, Gemma and L{\'o}pez-Rubio, Anna V and Pessina, Patrizia and Eder, Thomas and Shi, Junchao and Jen, Annie and Lupi{\'o}n-Garcia, N{\'u}ria and Starner, Anne C and Bartels, Mason D and Cui, Yingzhi and Sands, Caroline M and Planas-Riverola, Ainoa and Martínez, Alba and Velasco-Hernandez, Talia and Tomás-Daza, Laureano and Alber, Bernhard and Manhart, Gabriele and Mayer, Isabella Maria and Kollmann, Karoline and Fatica, Alessandro and Menendez, Pablo and Shishkova, Evgenia and Rau, Rachel E and Javierre, Biola M and Coon, Joshua and Chen, Qi and Van Nostrand, Eric L and Sardina, Jose L and Grebien, Florian and Stefano, Bruno Di}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature Cell Biology}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1--14}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Nature Publishing Group}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41556-024-01489-6}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.nature.com/articles/s41556-024-01489-6}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1038/s41556-024-01489-6" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1038/s41556-024-01489-6" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr h-100"><abbr class="badge">JCI insight</abbr></div> <div id="hernandez2024paternal" class="col-sm-8 h-100"> <div class="title">Paternal hypercholesterolemia elicits sex-specific exacerbation of atherosclerosis in offspring</div> <div class="author">Rebecca Hernandez*, Xiuchun Li, <b>Junchao Shi</b>, Tejasvi R Dave, <a href="https://tongzhoulab.org/" rel="external nofollow noopener" target="_blank">Tong Zhou</a>, <a href="http://qichen-lab.info/" rel="external nofollow noopener" target="_blank">Qi Chen</a>, and Changcheng Zhou#</div> <div class="periodical"> <em>JCI insight</em> 2024 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://insight.jci.org/articles/view/179291" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2024/hernandez2024paternal.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>Emerging studies suggest that various parental exposures affect offspring cardiovascular health, yet the specific mechanisms, particularly the influence of paternal cardiovascular disease (CVD) risk factors on offspring cardiovascular health, remain elusive. The present study explores how paternal hypercholesterolemia affects offspring atherosclerosis development using the LDL receptor-deficient (LDLR-/-) mouse model. We found that paternal high-cholesterol diet feeding led to significantly increased atherosclerosis in F1 female, but not male, LDLR-/- offspring. Transcriptomic analysis highlighted that paternal hypercholesterolemia stimulated proatherogenic genes, including Ccn1 and Ccn2, in the intima of female offspring. Sperm small noncoding RNAs (sncRNAs), particularly transfer RNA-derived (tRNA-derived) small RNAs (tsRNAs) and rRNA-derived small RNAs (rsRNAs), contribute to the intergenerational transmission of paternally acquired metabolic phenotypes. Using a newly developed PANDORA-Seq method, we identified that high-cholesterol feeding elicited changes in sperm tsRNA/rsRNA profiles that were undetectable by traditional RNA-Seq, and these altered sperm sncRNAs were potentially key factors mediating paternal hypercholesterolemia-elicited atherogenesis in offspring. Interestingly, high-cholesterol feeding altered sncRNA biogenesis-related gene expression in the epididymis but not testis of LDLR-/- sires; this may have led to the modified sperm sncRNA landscape. Our results underscore the sex-specific intergenerational effect of paternal hypercholesterolemia on offspring cardiovascular health and contribute to the understanding of chronic disease etiology originating from parental exposures.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">hernandez2024paternal</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Paternal hypercholesterolemia elicits sex-specific exacerbation of atherosclerosis in offspring}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Hernandez, Rebecca and Li, Xiuchun and Shi, Junchao and Dave, Tejasvi R and Zhou, Tong and Chen, Qi and Zhou, Changcheng}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{JCI insight}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{9}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{17}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{American Society for Clinical Investigation}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1172/jci.insight.179291}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://insight.jci.org/articles/view/179291}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1172/jci.insight.179291" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1172/jci.insight.179291" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li> </ol> <h2 class="year">2023</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr h-100"><abbr class="badge">Cell Biosci</abbr></div> <div id="ma2023mtor" class="col-sm-8 h-100"> <div class="title">mTOR hypoactivity leads to trophectoderm cell failure by enhancing lysosomal activation and disrupting the cytoskeleton in preimplantation embryo</div> <div class="author">Chiyuan Ma*, Qin Li*, Yuxin Yang, Lei Ge, Jiaxuan Cai, Juan Wang, Maoxian Zhu, Yue Xiong, Wenya Zhang, Jingtong Xie, Yujing Cao, Huashan Zhao, Qing Wei, Chen Huang, <b>Junchao Shi</b>, Jian V. Zhang, Enkui Duan, and Xiaohua Lei#</div> <div class="periodical"> <em>Cell &amp; Bioscience</em> Nov 2023 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://link.springer.com/article/10.1186/s13578-023-01176-3" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2023/ma2023mtor.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>Background: Metabolic homeostasis is closely related to early impairment of cell fate determination and embryo development. The protein kinase mechanistic target of rapamycin (mTOR) is a key regulator of cellular metabolism in the body. Inhibition of mTOR signaling in early embryo causes postimplantation development failure, yet the mechanisms are still poorly understood. Methods: Pregnancy mice and preimplantation mouse embryo were treated with mTOR inhibitor in vivo and in vitro respectively, and subsequently examined the blastocyst formation, implantation, and post-implantation development. We used immunofluorescence staining, RNA-Seq smart2, and genome-wide bisulfite sequencing technologies to investigate the impact of mTOR inhibitors on the quality, cell fate determination, and molecular alterations in developing embryos. Results: We showed mTOR suppression during preimplantation decreases the rate of blastocyst formation and the competency of implantation, impairs the post implantation embryonic development. We discovered that blocking mTOR signaling negatively affected the transformation of 8-cell embryos into blastocysts and caused various deficiencies in blastocyst quality. These included problems with compromised trophectoderm cell differentiation, as well as disruptions in cell fate specification. mTOR suppression significantly affected the transcription and DNA methylation of embryos. Treatment with mTOR inhibitors increase lysosomal activation and disrupts the organization and dynamics of the actin cytoskeleton in blastocysts.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">ma2023mtor</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{mTOR hypoactivity leads to trophectoderm cell failure by enhancing lysosomal activation and disrupting the cytoskeleton in preimplantation embryo}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Ma, Chiyuan and Li, Qin and Yang, Yuxin and Ge, Lei and Cai, Jiaxuan and Wang, Juan and Zhu, Maoxian and Xiong, Yue and Zhang, Wenya and Xie, Jingtong and Cao, Yujing and Zhao, Huashan and Wei, Qing and Huang, Chen and Shi, Junchao and Zhang, Jian V. and Duan, Enkui and Lei, Xiaohua}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Cell \&amp; Bioscience}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">nov</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Springer}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1186/s13578-023-01176-3}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://link.springer.com/article/10.1186/s13578-023-01176-3}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1186/s13578-023-01176-3" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1186/s13578-023-01176-3" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr h-100"><abbr class="badge">Cell Biosci</abbr></div> <div id="xue2023sex" class="col-sm-8 h-100"> <div class="title">Sex differences in paternal arsenic-induced intergenerational metabolic effects are mediated by estrogen</div> <div class="author">Yanfeng Xue, Yingyun Gong, Xin Li, Fei Peng, Guolian Ding, Zhao Zhang, <b>Junchao Shi</b>, Ilma Saleh Savul, Yong Xu, <a href="http://qichen-lab.info/" rel="external nofollow noopener" target="_blank">Qi Chen</a>, Leng Han, Shengyong Mao#, and Zheng Sun#</div> <div class="periodical"> <em>Cell &amp; Bioscience</em> Nov 2023 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://link.springer.com/article/10.1186/s13578-023-01121-4" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2023/xue2023sex.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>Background: Gene-environment interactions contribute to metabolic disorders such as diabetes and dyslipidemia. In addition to affecting metabolic homeostasis directly, drugs and environmental chemicals can cause persistent alterations in metabolic portfolios across generations in a sex-specific manner. Here, we use inorganic arsenic (iAs) as a prototype drug and chemical to dissect such sex differences. Methods: After weaning, C57BL/6 WT male mice were treated with 250 ppb iAs in drinking water (iAsF0) or normal water (conF0) for 6 weeks and then bred with 15-week-old, non-exposed females for 3 days in cages with only normal water (without iAs), to generate iAsF1 or conF1 mice, respectively. F0 females and all F1 mice drank normal water without iAs all the time. Results: We find that exposure of male mice to 250 ppb iAs leads to glucose intolerance and insulin resistance in F1 female offspring (iAsF1-F), with almost no change in blood lipid profiles. In contrast, F1 males (iAsF1-M) show lower liver and blood triglyceride levels than non-exposed control, with improved glucose tolerance and insulin sensitivity. The liver of F1 offspring shows sex-specific transcriptomic changes, with hepatocyte-autonomous alternations of metabolic fluxes in line with the sex-specific phenotypes. The iAsF1-F mice show altered levels of circulating estrogen and follicle-stimulating hormone. Ovariectomy or liver-specific knockout of estrogen receptor α/β made F1 females resemble F1 males in their metabolic responses to paternal iAs exposure. Conclusions: These results demonstrate that disrupted reproductive hormone secretion in alliance with hepatic estrogen signaling accounts for the sex-specific intergenerational effects of paternal iAs exposure, which shed light on the sex disparities in long-term gene-environment interactions.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">xue2023sex</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Sex differences in paternal arsenic-induced intergenerational metabolic effects are mediated by estrogen}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Xue, Yanfeng and Gong, Yingyun and Li, Xin and Peng, Fei and Ding, Guolian and Zhang, Zhao and Shi, Junchao and Savul, Ilma Saleh and Xu, Yong and Chen, Qi and Han, Leng and Mao, Shengyong and Sun, Zheng}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Cell \&amp; Bioscience}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{13}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{165}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Springer}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://link.springer.com/article/10.1186/s13578-023-01121-4}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr h-100"><abbr class="badge">Cell Res</abbr></div> <div id="zhang2023towards" class="col-sm-8 h-100"> <div class="title">Towards the understanding of "Herbal RNA Code" for traditional medicine</div> <div class="author">Ying Zhang#, <b>Junchao Shi</b>#, and <a href="http://qichen-lab.info/" rel="external nofollow noopener" target="_blank">Qi Chen</a> </div> <div class="periodical"> <em>Cell Research</em> Jul 2023 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.nature.com/articles/s41422-023-00851-x" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2023/zhang2023towards.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>Recent emerging studies have unveiled the presence of millions of herbal small RNAs (sRNAs) in decoction of traditional medicines, suggesting their roles in cross-kingdom communication. Investigating the origins of herbal sRNAs, their delivery mechanisms, and modes of action in alleviating diseases could pave the way for innovative pharmaceutical engineering and novel clinical treatments.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">zhang2023towards</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Towards the understanding of "Herbal RNA Code" for traditional medicine}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Zhang, Ying and Shi, Junchao and Chen, Qi}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Cell Research}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jul</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Nature Publishing Group}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41422-023-00851-x}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.nature.com/articles/s41422-023-00851-x}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1038/s41422-023-00851-x" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1038/s41422-023-00851-x" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li> </ol> <a name="works_at_UCR"></a> <h1>Works at UCR</h1> <h2 class="year">2023</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr h-100"> <img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/2023_JLR_cover.png"> <div class="w-100"></div> <abbr2 class="award badge">Cover Story</abbr2> </div> <div id="hernandez2023pandora" class="col-sm-8 h-100"> <div class="title">PANDORA-Seq unveils the hidden small noncoding RNA landscape in atherosclerosis of LDL receptor-deficient mice</div> <div class="author">Rebecca Hernandez*, <b>Junchao Shi</b>*, Jingwei Liu, Xiuchun Li, Jake Wu, Linlin Zhao, <a href="https://tongzhoulab.org/" rel="external nofollow noopener" target="_blank">Tong Zhou</a>, <a href="http://qichen-lab.info/" rel="external nofollow noopener" target="_blank">Qi Chen</a>, and Changcheng Zhou#</div> <div class="periodical"> <em>Journal of Lipid Research</em> Apr 2023 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.sciencedirect.com/science/article/pii/S0022227523000251" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2023/hernandez2023pandora.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>Small noncoding RNAs (sncRNAs) play diverse roles in numerous biological processes. While the widely used RNA sequencing (RNA-Seq) method has advanced sncRNA discovery, RNA modifications can interfere with the complementary DNA library construction process, preventing the discovery of highly modified sncRNAs including transfer RNA-derived small RNAs (tsRNAs) and ribosomal RNA-derived small RNAs (rsRNAs) that may have important functions in disease development. To address this technical obstacle, we recently developed a novel PANDORA-Seq (Panoramic RNA Display by Overcoming RNA Modification Aborted Sequencing) method to overcome RNA modification-elicited sequence interferences. To identify novel sncRNAs associated with atherosclerosis development, LDL receptor-deficient (LDLR−/−) mice were fed a low-cholesterol diet or high-cholesterol diet (HCD) for 9 weeks. Total RNAs isolated from the intima were subjected to PANDORA-Seq and traditional RNA-Seq. By overcoming RNA modification-elicited limitations, PANDORA-Seq unveiled an rsRNA/tsRNA-enriched sncRNA landscape in the atherosclerotic intima of LDLR−/− mice, which was strikingly different from that detected by traditional RNA-Seq. While microRNAs were the dominant sncRNAs detected by traditional RNA-Seq, PANDORA-Seq substantially increased the reads of rsRNAs and tsRNAs. PANDORA-Seq also detected 1,383 differentially expressed sncRNAs induced by HCD feeding, including 1,160 rsRNAs and 195 tsRNAs. One of HCD-induced intimal tsRNAs, tsRNA-Arg-CCG, may contribute to atherosclerosis development by regulating the proatherogenic gene expression in endothelial cells. Overall, PANDORA-Seq revealed a hidden rsRNA and tsRNA population associated with atherosclerosis development. These understudied tsRNAs and rsRNAs, which are much more abundant than microRNAs in the atherosclerotic intima of LDLR−/− mice, warrant further investigations.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">hernandez2023pandora</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{PANDORA-Seq unveils the hidden small noncoding RNA landscape in atherosclerosis of LDL receptor-deficient mice}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Hernandez, Rebecca and Shi, Junchao and Liu, Jingwei and Li, Xiuchun and Wu, Jake and Zhao, Linlin and Zhou, Tong and Chen, Qi and Zhou, Changcheng}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Journal of Lipid Research}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.jlr.2023.100352}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{64}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{4}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">apr</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Elsevier}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.sciencedirect.com/science/article/pii/S0022227523000251}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1016/j.jlr.2023.100352" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1016/j.jlr.2023.100352" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr h-100"><abbr class="badge">Environ Int</abbr></div> <div id="liu2023paternal" class="col-sm-8 h-100"> <div class="title">Paternal phthalate exposure-elicited offspring metabolic disorders are associated with altered sperm small RNAs in mice</div> <div class="author">Jingwei Liu, <b>Junchao Shi</b>, Rebecca Hernandez, Xiuchun Li, Pranav Konchadi, Yuma Miyake, <a href="http://qichen-lab.info/" rel="external nofollow noopener" target="_blank">Qi Chen</a>, <a href="https://tongzhoulab.org/" rel="external nofollow noopener" target="_blank">Tong Zhou</a>, and Changcheng Zhou#</div> <div class="periodical"> <em>Environment International</em> Feb 2023 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.sciencedirect.com/science/article/pii/S0160412023000429" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2023/liu2023paternal.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>Exposure to ubiquitous plastic-associated endocrine disrupting chemicals (EDCs) is associated with the increased risk of many chronic diseases. For example, phthalate exposure is associated with cardiometabolic mortality in humans, with societal costs ∼ $39 billion/year or more. We recently demonstrated that several widely used plastic-associated EDCs increase cardiometabolic disease in appropriate mouse models. In addition to affecting adult health, parental exposure to EDCs has also been shown to cause metabolic disorders, including obesity and diabetes, in the offspring. While most studies have focused on the impact of maternal EDC exposure on the offspring’s health, little is known about the effects of paternal EDC exposure. In the current study, we investigated the adverse impact of paternal exposure to a ubiquitous but understudied phthalate, dicyclohexyl phthalate (DCHP) on the metabolic health of F1 and F2 offspring in mice. Paternal DCHP exposure led to exacerbated insulin resistance and impaired insulin signaling in F1 offspring without affecting diet-induced obesity. We previously showed that sperm small non-coding RNAs including tRNA-derived small RNAs (tsRNAs) and rRNA-derived small RNAs (rsRNAs) contribute to the intergenerational transmission of paternally acquired metabolic disorders. Using a novel PANDORA-seq, we revealed that DCHP exposure can lead to sperm tsRNA/rsRNA landscape changes that were undetected by traditional RNA-seq, which may contribute to DCHP-elicited adverse effects. Lastly, we found that paternal DCHP can also cause sex-specific transgenerational adverse effects in F2 offspring and elicited glucose intolerance in female F2 descendants. Our results suggest that exposure to endocrine disrupting phthalates may have intergenerational and transgenerational adverse effects on the metabolic health of their offspring. These findings increase our understanding of the etiology of chronic human diseases originating from chemical-elicited intergenerational and transgenerational effects.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">liu2023paternal</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Paternal phthalate exposure-elicited offspring metabolic disorders are associated with altered sperm small RNAs in mice}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Liu, Jingwei and Shi, Junchao and Hernandez, Rebecca and Li, Xiuchun and Konchadi, Pranav and Miyake, Yuma and Chen, Qi and Zhou, Tong and Zhou, Changcheng}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Environment International}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{107769}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">feb</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Elsevier}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.envint.2023.107769}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.sciencedirect.com/science/article/pii/S0160412023000429}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1016/j.envint.2023.107769" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1016/j.envint.2023.107769" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li> </ol> <h2 class="year">2022</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr h-100"> <img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/2022_NCB_cover.png"> <div class="w-100"></div> <abbr2 class="award badge">Cover Story</abbr2> </div> <div id="shi2022exploring" class="col-sm-8 h-100"> <div class="title">Exploring the expanding universe of small RNAs</div> <div class="author"> <b>Junchao Shi</b>, <a href="https://tongzhoulab.org/" rel="external nofollow noopener" target="_blank">Tong Zhou</a>#, and <a href="http://qichen-lab.info/" rel="external nofollow noopener" target="_blank">Qi Chen</a># </div> <div class="periodical"> <em>Nature Cell Biology</em> Apr 2022 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.nature.com/articles/s41556-022-00880-5" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2022/shi2022exploring.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>The world of small noncoding RNAs (sncRNAs) is ever-expanding, from small interfering RNA, microRNA and Piwi-interacting RNA to the recently emerging non-canonical sncRNAs derived from longer structured RNAs (for example, transfer, ribosomal, Y, small nucleolar, small nuclear and vault RNAs), showing distinct biogenesis and functional principles. Here we discuss recent tools for sncRNA identification, caveats in sncRNA expression analysis and emerging methods for direct sequencing of sncRNAs and systematic mapping of RNA modifications that are integral to their function.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">shi2022exploring</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Exploring the expanding universe of small RNAs}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Shi, Junchao and Zhou, Tong and Chen, Qi}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature Cell Biology}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{24}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{4}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{415--423}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">apr</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Nature Publishing Group}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41556-022-00880-5}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.nature.com/articles/s41556-022-00880-5}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1038/s41556-022-00880-5" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1038/s41556-022-00880-5" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li></ol> <h2 class="year">2021</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr h-100"><abbr class="badge">Protein Cell</abbr></div> <div id="qu2021cooperation" class="col-sm-8 h-100"> <div class="title">Cooperation-based sperm clusters mediate sperm oviduct entry and fertilization</div> <div class="author">Yongcun Qu*, <a href="http://qichen-lab.info/" rel="external nofollow noopener" target="_blank">Qi Chen</a>*, Shanshan Guo*, Chiyuan Ma, Yonggang Lu, <b>Junchao Shi</b>, Shichao Liu, <a href="https://tongzhoulab.org/" rel="external nofollow noopener" target="_blank">Tong Zhou</a>, Taichi Noda, Jingjing Qian, Liwen Zhang, Xili Zhu, Xiaohua Lei, Yujing Cao, Wei Li, Nicolas Plachta, Martin M Matzuk, Masahito Ikawa, Enkui Duan#, Ying Zhang#, and Hongmei Wang#</div> <div class="periodical"> <em>Protein &amp; cell</em> Mar 2021 </div> <div class="links"> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://link.springer.com/article/10.1007/s13238-021-00825-y" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2021/qu2021cooperation.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">qu2021cooperation</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Cooperation-based sperm clusters mediate sperm oviduct entry and fertilization}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Qu, Yongcun and Chen, Qi and Guo, Shanshan and Ma, Chiyuan and Lu, Yonggang and Shi, Junchao and Liu, Shichao and Zhou, Tong and Noda, Taichi and Qian, Jingjing and Zhang, Liwen and Zhu, Xili and Lei, Xiaohua and Cao, Yujing and Li, Wei and Plachta, Nicolas and Matzuk, Martin M and Ikawa, Masahito and Duan, Enkui and Zhang, Ying and Wang, Hongmei}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Protein \&amp; cell}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{12}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{10}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{810--817}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">mar</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Springer Nature}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/s13238-021-00825-y}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://link.springer.com/article/10.1007/s13238-021-00825-y}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1007/s13238-021-00825-y" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1007/s13238-021-00825-y" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr h-100"><abbr class="badge">Nat Cell Biol</abbr></div> <div id="shi2021pandora" class="col-sm-8 h-100"> <div class="title">PANDORA-seq expands the repertoire of regulatory small RNAs by overcoming RNA modifications</div> <div class="author"> <b>Junchao Shi</b>*, Yunfang Zhang*, Dongmei Tan*, Xudong Zhang*, Menghong Yan*, Ying Zhang*, Reuben Franklin*, Marta Shahbazi, Kirsty Mackinlay, Shichao Liu, Bernhard Kuhle, Emma R James, Liwen Zhang, Yongcun Qu, Qiwei Zhai, Wenxin Zhao, Linlin Zhao, Changcheng Zhou, Weifeng Gu, Jernej Murn, Jingtao Guo, Douglas T Carrell, Yinsheng Wang, Xuemei Chen, Bradley R Cairns, Xiang-lei Yang, Paul Schimmel, <a href="https://mzglab.com/" rel="external nofollow noopener" target="_blank">Magdalena Zernicka-Goetz</a>, Sihem Cheloufi#, Ying Zhang#, <a href="https://tongzhoulab.org/" rel="external nofollow noopener" target="_blank">Tong Zhou</a>#, and <a href="http://qichen-lab.info/" rel="external nofollow noopener" target="_blank">Qi Chen</a># </div> <div class="periodical"> <em>Nature cell biology</em> Apr 2021 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.nature.com/articles/s41556-021-00652-7" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2021/shi2021pandora.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>Although high-throughput RNA sequencing (RNA-seq) has greatly advanced small non-coding RNA (sncRNA) discovery, the currently widely used complementary DNA library construction protocol generates biased sequencing results. This is partially due to RNA modifications that interfere with adapter ligation and reverse transcription processes, which prevent the detection of sncRNAs bearing these modifications. Here, we present PANDORA-seq (panoramic RNA display by overcoming RNA modification aborted sequencing), employing a combinatorial enzymatic treatment to remove key RNA modifications that block adapter ligation and reverse transcription. PANDORA-seq identified abundant modified sncRNAs-mostly transfer RNA-derived small RNAs (tsRNAs) and ribosomal RNA-derived small RNAs (rsRNAs)-that were previously undetected, exhibiting tissue-specific expression across mouse brain, liver, spleen and sperm, as well as cell-specific expression across embryonic stem cells (ESCs) and HeLa cells. Using PANDORA-seq, we revealed unprecedented landscapes of microRNA, tsRNA and rsRNA dynamics during the generation of induced pluripotent stem cells. Importantly, tsRNAs and rsRNAs that are downregulated during somatic cell reprogramming impact cellular translation in ESCs, suggesting a role in lineage differentiation.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">shi2021pandora</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{PANDORA-seq expands the repertoire of regulatory small RNAs by overcoming RNA modifications}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Shi, Junchao and Zhang, Yunfang and Tan, Dongmei and Zhang, Xudong and Yan, Menghong and Zhang, Ying and Franklin, Reuben and Shahbazi, Marta and Mackinlay, Kirsty and Liu, Shichao and Kuhle, Bernhard and James, Emma R and Zhang, Liwen and Qu, Yongcun and Zhai, Qiwei and Zhao, Wenxin and Zhao, Linlin and Zhou, Changcheng and Gu, Weifeng and Murn, Jernej and Guo, Jingtao and Carrell, Douglas T and Wang, Yinsheng and Chen, Xuemei and Cairns, Bradley R and Yang, Xiang-lei and Schimmel, Paul and Zernicka-Goetz, Magdalena and Cheloufi, Sihem and Zhang, Ying and Zhou, Tong and Chen, Qi}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature cell biology}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{23}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{4}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{424--436}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">apr</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Nature Publishing Group}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41556-021-00652-7}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.nature.com/articles/s41556-021-00652-7}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1038/s41556-021-00652-7" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1038/s41556-021-00652-7" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr h-100"><abbr class="badge">Trends Biochem Sci</abbr></div> <div id="chen2021origins" class="col-sm-8 h-100"> <div class="title">Origins and evolving functionalities of tRNA-derived small RNAs</div> <div class="author"> <a href="http://qichen-lab.info/" rel="external nofollow noopener" target="_blank">Qi Chen</a>#, Xudong Zhang, <b>Junchao Shi</b>, Menghong Yan, and <a href="https://tongzhoulab.org/" rel="external nofollow noopener" target="_blank">Tong Zhou</a># </div> <div class="periodical"> <em>Trends in biochemical sciences</em> Oct 2021 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.sciencedirect.com/science/article/pii/S0968000421001031" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2021/chen2021origins.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>Transfer RNA (tRNA)-derived small RNAs (tsRNAs) are among the most ancient small RNAs in all domains of life and are generated by the cleavage of tRNAs. Emerging studies have begun to reveal the versatile roles of tsRNAs in fundamental biological processes, including gene silencing, ribosome biogenesis, retrotransposition, and epigenetic inheritance, which are rooted in tsRNA sequence conservation, RNA modifications, and protein-binding abilities. We summarize the mechanisms of tsRNA biogenesis and the impact of RNA modifications, and propose how thinking of tsRNA functionality from an evolutionary perspective urges the expansion of tsRNA research into a wider spectrum, including cross-tissue/cross-species regulation and harnessing of the ’tsRNA code’ for precision medicine.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">chen2021origins</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Origins and evolving functionalities of tRNA-derived small RNAs}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Chen, Qi and Zhang, Xudong and Shi, Junchao and Yan, Menghong and Zhou, Tong}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Trends in biochemical sciences}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{46}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{10}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{790--804}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">oct</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Elsevier Current Trends}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.tibs.2021.05.001}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.sciencedirect.com/science/article/pii/S0968000421001031}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1016/j.tibs.2021.05.001" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1016/j.tibs.2021.05.001" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr h-100"><abbr class="badge">Ann N Y Acad Sci</abbr></div> <div id="cable2021noncoding" class="col-sm-8 h-100"> <div class="title">Noncoding RNAs: biology and applications-a Keystone Symposia report</div> <div class="author">Jennifer Cable#, Edith Heard, Tetsuro Hirose, Kannanganattu V Prasanth, Ling-Ling Chen, Jonathan E Henninger, Sofia A Quinodoz, David L Spector, Sarah D Diermeier, Allison M Porman, Dhiraj Kumar, Mark W Feinberg, Xiaohua Shen, Juan Pablo Unfried, Rory Johnson, Chun-Kan Chen, Jeremy Wilusz, E Adelheid Lempradl, Sean E McGeary, Lamia Wahba, Anna Marie Pyle, Amanda E Hargrove, Matthew D Simon, Marco Marcia, Róża K Przanowska, Howard Y Chang, Samie R Jaffrey, Lydia M Contreras, <a href="http://qichen-lab.info/" rel="external nofollow noopener" target="_blank">Qi Chen</a>, <b>Junchao Shi</b>, Joshua T Mendell, Lin He, Erwei Song, John L Rinn, Mukesh Kumar Lalwani, Murat Can Kalem, Edward B Chuong, Lynne E Maquat, and Xuhang Liu</div> <div class="periodical"> <em>Annals of the New York Academy of Sciences</em> Dec 2021 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://nyaspubs.onlinelibrary.wiley.com/doi/10.1111/nyas.14713" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2021/cable2021noncoding.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>The human transcriptome contains many types of noncoding RNAs, which rival the number of protein-coding species. From long noncoding RNAs (lncRNAs) that are over 200 nucleotides long to piwi-interacting RNAs (piRNAs) of only 20 nucleotides, noncoding RNAs play important roles in regulating transcription, epigenetic modifications, translation, and cell signaling. Roles for noncoding RNAs in disease mechanisms are also being uncovered, and several species have been identified as potential drug targets. On May 11-14, 2021, the Keystone eSymposium "Noncoding RNAs: Biology and Applications" brought together researchers working in RNA biology, structure, and technologies to accelerate both the understanding of RNA basic biology and the translation of those findings into clinical applications.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">cable2021noncoding</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Noncoding RNAs: biology and applications-a Keystone Symposia report}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Cable, Jennifer and Heard, Edith and Hirose, Tetsuro and Prasanth, Kannanganattu V and Chen, Ling-Ling and Henninger, Jonathan E and Quinodoz, Sofia A and Spector, David L and Diermeier, Sarah D and Porman, Allison M and Kumar, Dhiraj and Feinberg, Mark W and Shen, Xiaohua and Unfried, Juan Pablo and Johnson, Rory and Chen, Chun-Kan and Wilusz, Jeremy and Lempradl, E Adelheid and McGeary, Sean E and Wahba, Lamia and Pyle, Anna Marie and Hargrove, Amanda E and Simon, Matthew D and Marcia, Marco and Przanowska, Róża K and Chang, Howard Y and Jaffrey, Samie R and Contreras, Lydia M and Chen, Qi and Shi, Junchao and Mendell, Joshua T and He, Lin and Song, Erwei and Rinn, John L and Lalwani, Mukesh Kumar and Kalem, Murat Can and Chuong, Edward B and Maquat, Lynne E and Liu, Xuhang}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Annals of the New York Academy of Sciences}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{1506}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{118--141}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">dec</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1111/nyas.14713}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://nyaspubs.onlinelibrary.wiley.com/doi/10.1111/nyas.14713}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1111/nyas.14713" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1111/nyas.14713" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li> </ol> <h2 class="year">2020</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr h-100"><abbr class="badge">Natl Sci Rev</abbr></div> <div id="lei2020development" class="col-sm-8 h-100"> <div class="title">Development of mouse preimplantation embryos in space</div> <div class="author">Xiaohua Lei*#, Yujing Cao*, Baohua Ma*, Yunfang Zhang, Lina Ning, Jingjing Qian, Liwen Zhang, Yongcun Qu, Tao Zhang, Dehong Li, <a href="http://qichen-lab.info/" rel="external nofollow noopener" target="_blank">Qi Chen</a>, <b>Junchao Shi</b>, Xudong Zhang, Chiyuan Ma, Ying Zhang#, and Enkui Duan#</div> <div class="periodical"> <em>National Science Review</em> Sep 2020 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://academic.oup.com/nsr/article/7/9/1437/5819033" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2020/lei2020development.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>The development of life beyond planet Earth is a long-standing quest of the human race, but whether normal mammalian embryonic development can occur in space is still unclear. Here, we show unequivocally that preimplantation mouse embryos can develop in space, but the rate of blastocyst formation and blastocyst quality are compromised. Additionally, the cells in the embryo contain severe DNA damage, while the genome of the blastocysts developed in space is globally hypomethylated with a unique set of differentially methylated regions. The developmental defects, DNA damage and epigenetic abnormalities can be largely mimicked by the treatment with ground-based low-dose radiation. However, the exposure to simulated microgravity alone does not cause major disruptions of embryonic development, indicating that radiation is the main cause for the developmental defects. This work advances the understanding of embryonic development in space and reveals long-term extreme low-dose radiation as a hazardous factor for mammalian reproduction.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">lei2020development</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Development of mouse preimplantation embryos in space}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Lei, Xiaohua and Cao, Yujing and Ma, Baohua and Zhang, Yunfang and Ning, Lina and Qian, Jingjing and Zhang, Liwen and Qu, Yongcun and Zhang, Tao and Li, Dehong and Chen, Qi and Shi, Junchao and Zhang, Xudong and Ma, Chiyuan and Zhang, Ying and Duan, Enkui}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{National Science Review}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{7}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{9}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1437--1446}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">sep</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Oxford University Press}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1093/nsr/nwaa062}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://academic.oup.com/nsr/article/7/9/1437/5819033}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1093/nsr/nwaa062" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1093/nsr/nwaa062" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr h-100"><abbr class="badge">Mol Cancer</abbr></div> <div id="gu2020peripheral" class="col-sm-8 h-100"> <div class="title">Peripheral blood non-canonical small non-coding RNAs as novel biomarkers in lung cancer</div> <div class="author">Wanjun Gu*#, <b>Junchao Shi</b>*, Hui Liu*, Xudong Zhang*, Jin J Zhou, Musheng Li, Dandan Zhou, Rui Li, Jingzhu Lv, Guoxia Wen, Shanshan Zhu, Ting Qi, Wei Li, Xiaojing Wang, Zhaohua Wang, Hua Zhu, Changcheng Zhou, Kenneth S Knox, Ting Wang, <a href="http://qichen-lab.info/" rel="external nofollow noopener" target="_blank">Qi Chen</a>#, Zhongqing Qian#, and <a href="https://tongzhoulab.org/" rel="external nofollow noopener" target="_blank">Tong Zhou</a># </div> <div class="periodical"> <em>Molecular cancer</em> Nov 2020 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://molecular-cancer.biomedcentral.com/articles/10.1186/s12943-020-01280-9" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2020/gu2020peripheral.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>One unmet challenge in lung cancer diagnosis is to accurately differentiate lung cancer from other lung diseases with similar clinical symptoms and radiological features, such as pulmonary tuberculosis (TB). To identify reliable biomarkers for lung cancer screening, we leverage the recently discovered non-canonical small non-coding RNAs (i.e., tRNA-derived small RNAs [tsRNAs], rRNA-derived small RNAs [rsRNAs], and YRNA-derived small RNAs [ysRNAs]) in human peripheral blood mononuclear cells and develop a molecular signature composed of distinct ts/rs/ysRNAs (TRY-RNA). Our TRY-RNA signature precisely discriminates between control, lung cancer, and pulmonary TB subjects in both the discovery and validation cohorts and outperforms microRNA-based biomarkers, which bears the diagnostic potential for lung cancer screening.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">gu2020peripheral</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Peripheral blood non-canonical small non-coding RNAs as novel biomarkers in lung cancer}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Gu, Wanjun and Shi, Junchao and Liu, Hui and Zhang, Xudong and Zhou, Jin J and Li, Musheng and Zhou, Dandan and Li, Rui and Lv, Jingzhu and Wen, Guoxia and Zhu, Shanshan and Qi, Ting and Li, Wei and Wang, Xiaojing and Wang, Zhaohua and Zhu, Hua and Zhou, Changcheng and Knox, Kenneth S and Wang, Ting and Chen, Qi and Qian, Zhongqing and Zhou, Tong}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Molecular cancer}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{19}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1--6}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">nov</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{BioMed Central}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1186/s12943-020-01280-9}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://molecular-cancer.biomedcentral.com/articles/10.1186/s12943-020-01280-9}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1186/s12943-020-01280-9" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1186/s12943-020-01280-9" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr h-100"><abbr class="badge">GPB</abbr></div> <div id="wang2020denoising" class="col-sm-8 h-100"> <div class="title">Denoising autoencoder, a deep learning algorithm, aids the identification of a novel molecular signature of lung adenocarcinoma</div> <div class="author">Jun Wang*, Xueying Xie*, <b>Junchao Shi</b>, Wenjun He, <a href="http://qichen-lab.info/" rel="external nofollow noopener" target="_blank">Qi Chen</a>, Liang Chen#, Wanjun Gu#, and <a href="https://tongzhoulab.org/" rel="external nofollow noopener" target="_blank">Tong Zhou</a># </div> <div class="periodical"> <em>Genomics, proteomics &amp; bioinformatics</em> Aug 2020 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.sciencedirect.com/science/article/pii/S1672022920301376" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2020/wang2020denoising.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>Precise biomarker development is a key step in disease management. However, most of the published biomarkers were derived from a relatively small number of samples with supervised approaches. Recent advances in unsupervised machine learning promise to leverage very large datasets for making better predictions of disease biomarkers. Denoising autoencoder (DA) is one of the unsupervised deep learning algorithms, which is a stochastic version of autoencoder techniques. The principle of DA is to force the hidden layer of autoencoder to capture more robust features by reconstructing a clean input from a corrupted one. Here, a DA model was applied to analyze integrated transcriptomic data from 13 published lung cancer studies, which consisted of 1916 human lung tissue samples. Using DA, we discovered a molecular signature composed of multiple genes for lung adenocarcinoma (ADC). In independent validation cohorts, the proposed molecular signature is proved to be an effective classifier for lung cancer histological subtypes. Also, this signature successfully predicts clinical outcome in lung ADC, which is independent of traditional prognostic factors. More importantly, this signature exhibits a superior prognostic power compared with the other published prognostic genes. Our study suggests that unsupervised learning is helpful for biomarker development in the era of precision medicine.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">wang2020denoising</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Wang, Jun and Xie, Xueying and Shi, Junchao and He, Wenjun and Chen, Qi and Chen, Liang and Gu, Wanjun and Zhou, Tong}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Genomics, proteomics \&amp; bioinformatics}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{18}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{4}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{468--480}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">aug</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Elsevier}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.gpb.2019.02.003}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.sciencedirect.com/science/article/pii/S1672022920301376}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1016/j.gpb.2019.02.003" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1016/j.gpb.2019.02.003" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr h-100"><abbr class="badge">Neurobiol Dis</abbr></div> <div id="zhang2020small" class="col-sm-8 h-100"> <div class="title">Small RNA modifications in Alzheimer’s disease</div> <div class="author">Xudong Zhang*, Fatima Trebak*, Lucas AC Souza, <b>Junchao Shi</b>, <a href="https://tongzhoulab.org/" rel="external nofollow noopener" target="_blank">Tong Zhou</a>, Patrick G Kehoe, <a href="http://qichen-lab.info/" rel="external nofollow noopener" target="_blank">Qi Chen</a>#, and Yumei Feng Earley#</div> <div class="periodical"> <em>Neurobiology of disease</em> Aug 2020 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.sciencedirect.com/science/article/pii/S0969996120303338" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2020/zhang2020small.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>Background While significant advances have been made in uncovering the aetiology of Alzheimer’s disease and related dementias at the genetic level, molecular events at the epigenetic level remain largely undefined. Emerging evidence indicates that small non-coding RNAs (sncRNAs) and their associated RNA modifications are important regulators of complex physiological and pathological processes, including aging, stress responses, and epigenetic inheritance. However, whether small RNAs and their modifications are altered in dementia is not known. Methods We performed LC-MS/MS-based, high-throughput assays of small RNA modifications in post-mortem samples of the prefrontal lobe cortices of Alzheimer’s disease (AD) and control individuals. We noted that some of the AD patients has co-occurring vascular cognitive impairment-related pathology (VaD). Findings We report altered small RNA modifications in AD samples compared with normal controls. The 15-25-nucleotide (nt) RNA fraction of these samples was enriched for microRNAs, whereas the 30-40-nt RNA fraction was enriched for tRNA-derived small RNAs (tsRNAs), rRNA-derived small RNAs (rsRNAs), and YRNA-derived small RNAs (ysRNAs). Interestingly, most of these altered RNA modifications were detected both in the AD and AD with co-occurring vascular dementia subjects. In addition, sequencing of small RNA in the 30-40-nt fraction from AD cortices revealed reductions in rsRNA-5S, tsRNA-Tyr, and tsRNA-Arg. Interpretation These data suggest that sncRNAs and their associated modifications are novel signals that may be linked to the pathogenesis and development of Alzheimer’s disease.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">zhang2020small</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Small RNA modifications in Alzheimer's disease}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Zhang, Xudong and Trebak, Fatima and Souza, Lucas AC and Shi, Junchao and Zhou, Tong and Kehoe, Patrick G and Chen, Qi and Earley, Yumei Feng}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Neurobiology of disease}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{145}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{105058}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Academic Press}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.nbd.2020.105058}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.sciencedirect.com/science/article/pii/S0969996120303338}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1016/j.nbd.2020.105058" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1016/j.nbd.2020.105058" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li> </ol> <a name="works_at_UNR"></a> <h1>Works at UNR</h1> <h2 class="year">2019</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr h-100"><abbr class="badge">Trends Biochem Sci</abbr></div> <div id="shi2019tsrnas" class="col-sm-8 h-100"> <div class="title">tsRNAs: the Swiss army knife for translational regulation</div> <div class="author"> <b>Junchao Shi</b>*, Yunfang Zhang*, <a href="https://tongzhoulab.org/" rel="external nofollow noopener" target="_blank">Tong Zhou</a>#, and <a href="http://qichen-lab.info/" rel="external nofollow noopener" target="_blank">Qi Chen</a># </div> <div class="periodical"> <em>Trends in biochemical sciences</em> Mar 2019 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.sciencedirect.com/science/article/pii/S0968000418301907" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2019/shi2019tsrnas.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>tRNA-derived small RNAs (tsRNAs, or tRFs) are a new category of regulatory noncoding RNAs with versatile functions. Recent emerging studies have begun to unveil distinct features of tsRNAs based on their sequence, RNA modifications, and structures that differentially impact their functions towards regulating multiple aspects of translational control and ribosome biogenesis.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">shi2019tsrnas</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{tsRNAs: the Swiss army knife for translational regulation}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Shi, Junchao and Zhang, Yunfang and Zhou, Tong and Chen, Qi}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Trends in biochemical sciences}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{44}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{3}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{185--189}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2019}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">mar</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Elsevier Current Trends}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.tibs.2018.09.007}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.sciencedirect.com/science/article/pii/S0968000418301907}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1016/j.tibs.2018.09.007" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1016/j.tibs.2018.09.007" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr h-100"><abbr class="badge">Nat Rev Endocrinol</abbr></div> <div id="zhang2019sperm" class="col-sm-8 h-100"> <div class="title">Sperm RNA code programmes the metabolic health of offspring</div> <div class="author">Yunfang Zhang*, <b>Junchao Shi</b>*, Minoo Rassoulzadegan, Francesca Tuorto, and <a href="http://qichen-lab.info/" rel="external nofollow noopener" target="_blank">Qi Chen</a># </div> <div class="periodical"> <em>Nature Reviews Endocrinology</em> Aug 2019 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.nature.com/articles/s41574-019-0226-2" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2019/zhang2019sperm.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>Mammalian sperm RNA is increasingly recognized as an additional source of paternal hereditary information beyond DNA. Environmental inputs, including an unhealthy diet, mental stresses and toxin exposure, can reshape the sperm RNA signature and induce offspring phenotypes that relate to paternal environmental stressors. Our understanding of the categories of sperm RNAs (such as tRNA-derived small RNAs, microRNAs, ribosomal RNA-derived small RNAs and long non-coding RNAs) and associated RNA modifications is expanding and has begun to reveal the functional diversity and information capacity of these molecules. However, the coding mechanism endowed by sperm RNA structures and by RNA interactions with DNA and other epigenetic factors remains unknown. How sperm RNA-encoded information is decoded in early embryos to control offspring phenotypes also remains unclear. Complete deciphering of the ’sperm RNA code’ with regard to metabolic control could move the field towards translational applications and precision medicine, and this may lead to prevention of intergenerational transmission of obesity and type 2 diabetes mellitus susceptibility.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">zhang2019sperm</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Zhang, Yunfang and Shi, Junchao and Rassoulzadegan, Minoo and Tuorto, Francesca and Chen, Qi}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature Reviews Endocrinology}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{15}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{8}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{489--498}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2019}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">aug</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Nature Publishing Group}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41574-019-0226-2}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.nature.com/articles/s41574-019-0226-2}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1038/s41574-019-0226-2" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1038/s41574-019-0226-2" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li> </ol> <h2 class="year">2018</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr h-100"><abbr class="badge">EBioMedicine</abbr></div> <div id="qian2018potential" class="col-sm-8 h-100"> <div class="title">Potential diagnostic power of blood circular RNA expression in active pulmonary tuberculosis</div> <div class="author">Zhongqing Qian, Hui Liu, Musheng Li, <b>Junchao Shi</b>, Na Li, Yao Zhang, Xiaojie Zhang, Jingzhu Lv, Xueying Xie, Yunfei Bai, Qinyu Ge, Eun-A Ko, Haiyang Tang, Ting Wang, Xiaojing Wang, Zhaohua Wang, <a href="https://tongzhoulab.org/" rel="external nofollow noopener" target="_blank">Tong Zhou</a>#, and Wanjun Gu#</div> <div class="periodical"> <em>EBioMedicine</em> Jan 2018 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.sciencedirect.com/science/article/pii/S2352396417304887" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2018/qian2018potential.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>Circular RNAs (circRNAs) are a class of novel RNAs with important biological functions, and aberrant expression of circRNAs has been implicated in human diseases. However, the feasibility of using blood circRNAs as disease biomarkers is largely unknown. We explored the potential of using human peripheral blood mononuclear cell (PBMC) circRNAs as marker molecules to diagnose active pulmonary tuberculosis (TB). First, we demonstrated that circRNAs are widely expressed in human PBMCs and that many are abundant enough to be detected. Second, we found that the magnitude of PBMC circRNAs in TB patients was higher than that in the paired healthy controls. Compared with host linear transcripts, the circRNAs within several pathways are disproportionately upregulated in active TB patients, including "Cytokine-cytokine receptor interaction", "Chemokine signaling pathway", "Neurotrophin signaling pathway", and "Bacterial invasion of epithelial cells". Based on the differentially expressed circRNAs within these pathways, we developed a PBMC circRNA-based molecular signature differentiating active TB patients from healthy controls. We validated the classification power of the PBMC circRNA signature in an independent cohort with the area under the receiver operating characteristic curve (AUC) at 0.946. Our results suggest that PBMC circRNAs are potentially reliable marker molecules in TB diagnosis.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">qian2018potential</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Potential diagnostic power of blood circular RNA expression in active pulmonary tuberculosis}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Qian, Zhongqing and Liu, Hui and Li, Musheng and Shi, Junchao and Li, Na and Zhang, Yao and Zhang, Xiaojie and Lv, Jingzhu and Xie, Xueying and Bai, Yunfei and Ge, Qinyu and Ko, Eun-A and Tang, Haiyang and Wang, Ting and Wang, Xiaojing and Wang, Zhaohua and Zhou, Tong and Gu, Wanjun}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{EBioMedicine}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{27}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{18--26}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2018}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jan</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Elsevier}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.ebiom.2017.12.007}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.sciencedirect.com/science/article/pii/S2352396417304887}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1016/j.ebiom.2017.12.007" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1016/j.ebiom.2017.12.007" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr h-100"><abbr class="badge">GPB</abbr></div> <div id="shi2018sports1" class="col-sm-8 h-100"> <div class="title">SPORTS1. 0: a tool for annotating and profiling non-coding RNAs optimized for rRNA-and tRNA-derived small RNAs</div> <div class="author"> <b>Junchao Shi</b>#, Eun-A Ko, Kenton M Sanders, <a href="http://qichen-lab.info/" rel="external nofollow noopener" target="_blank">Qi Chen</a>#, and <a href="https://tongzhoulab.org/" rel="external nofollow noopener" target="_blank">Tong Zhou</a># </div> <div class="periodical"> <em>Genomics, proteomics &amp; bioinformatics</em> Apr 2018 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.sciencedirect.com/science/article/pii/S1672022918300445" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2018/shi2018sports1.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> <a href="https://github.com/junchaoshi/sports1.0" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> </div> <div class="abstract hidden"> <p>High-throughput RNA-seq has revolutionized the process of small RNA (sRNA) discovery, leading to a rapid expansion of sRNA categories. In addition to the previously well-characterized sRNAs such as microRNAs (miRNAs), piwi-interacting RNAs (piRNAs), and small nucleolar RNA (snoRNAs), recent emerging studies have spotlighted on tRNA-derived sRNAs (tsRNAs) and rRNA-derived sRNAs (rsRNAs) as new categories of sRNAs that bear versatile functions. Since existing software and pipelines for sRNA annotation are mostly focused on analyzing miRNAs or piRNAs, here we developed the sRNA annotation pipelineoptimized for rRNA- and tRNA-derived sRNAs (SPORTS1.0). SPORTS1.0 is optimized for analyzing tsRNAs and rsRNAs from sRNA-seq data, in addition to its capacity to annotate canonical sRNAs such as miRNAs and piRNAs. Moreover, SPORTS1.0 can predict potential RNA modification sites based on nucleotide mismatches within sRNAs. SPORTS1.0 is precompiled to annotate sRNAs for a wide range of 68 species across bacteria, yeast, plant, and animal kingdoms, while additional species for analyses could be readily expanded upon end users’ input. For demonstration, by analyzing sRNA datasets using SPORTS1.0, we reveal that distinct signatures are present in tsRNAs and rsRNAs from different mouse cell types. We also find that compared to other sRNA species, tsRNAs bear the highest mismatch rate, which is consistent with their highly modified nature. SPORTS1.0 is an open-source software and can be publically accessed at https://github.com/junchaoshi/sports1.0.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">shi2018sports1</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{SPORTS1. 0: a tool for annotating and profiling non-coding RNAs optimized for rRNA-and tRNA-derived small RNAs}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Shi, Junchao and Ko, Eun-A and Sanders, Kenton M and Chen, Qi and Zhou, Tong}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Genomics, proteomics \&amp; bioinformatics}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{16}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{2}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{144--151}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2018}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">apr</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Elsevier}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.gpb.2018.04.004}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.sciencedirect.com/science/article/pii/S1672022918300445}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1016/j.gpb.2018.04.004" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1016/j.gpb.2018.04.004" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr h-100"><abbr class="badge">Nat Cell Biol</abbr></div> <div id="zhang2018dnmt2" class="col-sm-8 h-100"> <div class="title">Dnmt2 mediates intergenerational transmission of paternally acquired metabolic disorders through sperm small non-coding RNAs</div> <div class="author">Yunfang Zhang*, Xudong Zhang*, <b>Junchao Shi</b>*, Francesca Tuorto*, Xin Li*, Yusheng Liu, Reinhard Liebers, Liwen Zhang, Yongcun Qu, Jingjing Qian, Maya Pahima, Ying Liu, Menghong Yan, Zhonghong Cao, Xiaohua Lei, Yujing Cao, Hongying Peng, Shichao Liu, Yue Wang, Huili Zheng, Rebekah Woolsey, David Quilici, Qiwei Zhai, Lei Li, <a href="https://tongzhoulab.org/" rel="external nofollow noopener" target="_blank">Tong Zhou</a>, Wei Yan, Frank Lyko, Ying Zhang#, Qi Zhou#, Enkui Duan#, and <a href="http://qichen-lab.info/" rel="external nofollow noopener" target="_blank">Qi Chen</a># </div> <div class="periodical"> <em>Nature cell biology</em> May 2018 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.nature.com/articles/s41556-018-0087-2" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2018/zhang2018dnmt2.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>The discovery of RNAs (for example, messenger RNAs, non-coding RNAs) in sperm has opened the possibility that sperm may function by delivering additional paternal information aside from solely providing the DNA1. Increasing evidence now suggests that sperm small non-coding RNAs (sncRNAs) can mediate intergenerational transmission of paternally acquired phenotypes, including mental stress2,3 and metabolic disorders4,5,6. How sperm sncRNAs encode paternal information remains unclear, but the mechanism may involve RNA modifications. Here we show that deletion of a mouse tRNA methyltransferase, DNMT2, abolished sperm sncRNA-mediated transmission of high-fat-diet-induced metabolic disorders to offspring. Dnmt2 deletion prevented the elevation of RNA modifications (m5C, m2G) in sperm 30-40 nt RNA fractions that are induced by a high-fat diet. Also, Dnmt2 deletion altered the sperm small RNA expression profile, including levels of tRNA-derived small RNAs and rRNA-derived small RNAs, which might be essential in composing a sperm RNA ’coding signature’ that is needed for paternal epigenetic memory. Finally, we show that Dnmt2-mediated m5C contributes to the secondary structure and biological properties of sncRNAs, implicating sperm RNA modifications as an additional layer of paternal hereditary information.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">zhang2018dnmt2</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Dnmt2 mediates intergenerational transmission of paternally acquired metabolic disorders through sperm small non-coding RNAs}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Zhang, Yunfang and Zhang, Xudong and Shi, Junchao and Tuorto, Francesca and Li, Xin and Liu, Yusheng and Liebers, Reinhard and Zhang, Liwen and Qu, Yongcun and Qian, Jingjing and Pahima, Maya and Liu, Ying and Yan, Menghong and Cao, Zhonghong and Lei, Xiaohua and Cao, Yujing and Peng, Hongying and Liu, Shichao and Wang, Yue and Zheng, Huili and Woolsey, Rebekah and Quilici, David and Zhai, Qiwei and Li, Lei and Zhou, Tong and Yan, Wei and Lyko, Frank and Zhang, Ying and Zhou, Qi and Duan, Enkui and Chen, Qi}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature cell biology}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{20}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{5}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{535--540}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2018}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">may</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Nature Publishing Group}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41556-018-0087-2}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.nature.com/articles/s41556-018-0087-2}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1038/s41556-018-0087-2" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1038/s41556-018-0087-2" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr h-100"><abbr class="badge">Nat Commun</abbr></div> <div id="chen2018tracing" class="col-sm-8 h-100"> <div class="title">Tracing the origin of heterogeneity and symmetry breaking in the early mammalian embryo</div> <div class="author"> <a href="http://qichen-lab.info/" rel="external nofollow noopener" target="_blank">Qi Chen</a>*, <b>Junchao Shi</b>*, Yi Tao, and <a href="https://mzglab.com/" rel="external nofollow noopener" target="_blank">Magdalena Zernicka-Goetz</a># </div> <div class="periodical"> <em>Nature communications</em> May 2018 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.nature.com/articles/s41467-018-04155-2" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2018/chen2018tracing.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>A fundamental question in developmental and stem cell biology concerns the origin and nature of signals that initiate asymmetry leading to pattern formation and self-organization. Instead of having prominent pre-patterning determinants as present in model organisms (worms, sea urchin, frog), we propose that the mammalian embryo takes advantage of more subtle cues such as compartmentalized intracellular reactions that generate micro-scale inhomogeneity, which is gradually amplified over several cellular generations to drive pattern formation while keeping developmental plasticity. It is therefore possible that by making use of compartmentalized information followed by its amplification, mammalian embryos would follow general principle of development found in other organisms in which the spatial cue is more robustly presented.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">chen2018tracing</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Tracing the origin of heterogeneity and symmetry breaking in the early mammalian embryo}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Chen, Qi and Shi, Junchao and Tao, Yi and Zernicka-Goetz, Magdalena}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature communications}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{9}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1--11}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2018}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">may</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Nature Publishing Group}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41467-018-04155-2}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.nature.com/articles/s41467-018-04155-2}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1038/s41467-018-04155-2" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1038/s41467-018-04155-2" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr h-100"><abbr class="badge">Biol Reprod</abbr></div> <div id="qian2018caffeine" class="col-sm-8 h-100"> <div class="title">Caffeine consumption during early pregnancy impairs oviductal embryo transport, embryonic development and uterine receptivity in mice</div> <div class="author">Jingjing Qian, Yunfang Zhang, Yongcun Qu, Liwen Zhang, <b>Junchao Shi</b>, Xudong Zhang, Shichao Liu, Bo Hyun Kim, Sung Jin Hwang, <a href="https://tongzhoulab.org/" rel="external nofollow noopener" target="_blank">Tong Zhou</a>, <a href="http://qichen-lab.info/" rel="external nofollow noopener" target="_blank">Qi Chen</a>, Sean M Ward, Enkui Duan#, and Ying Zhang#</div> <div class="periodical"> <em>Biology of reproduction</em> Dec 2018 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://academic.oup.com/biolreprod/article/99/6/1266/5049471" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2018/qian2018caffeine.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>Caffeine consumption has been widely used as a central nervous system stimulant. Epidemiological studies, however, have suggested that maternal caffeine exposure during pregnancy is associated with increased abnormalities, including decreased fertility, delayed conception, early spontaneous abortions, and low birth weight. The mechanisms underlying the negative outcomes of caffeine consumption, particularly during early pregnancy, remain unclear. In present study, we found that pregnant mice treated with moderate (5 mg/kg) or high (30 mg/kg) dosage of caffeine (intraperitoneally or orally) during preimplantation resulted in retention of early embryos in the oviduct, defective embryonic development, and impaired embryo implantation. Transferring normal blastocysts into the uteri of caffeine-treated pseudopregnant females also showed abnormal embryo implantation, thus indicating impaired uterine receptivity by caffeine administration. The remaining embryos that managed to implant after caffeine treatment also showed increased embryo resorption rate and abnormal development at mid-term stage, and decreased weight at birth. In addition to a dose-dependent effect, significant variations between individual mice under the same caffeine dosage were also observed, suggesting different sensitivities to caffeine, similar to that observed in human populations. Collectively, our data revealed that caffeine exposure during early pregnancy impaired oviductal embryo transport, embryonic development, and uterine receptivity, which are responsible for abnormal implantation and pregnancy loss. The study raises the concern of caffeine consumption during early stages of pregnancy.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">qian2018caffeine</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Caffeine consumption during early pregnancy impairs oviductal embryo transport, embryonic development and uterine receptivity in mice}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Qian, Jingjing and Zhang, Yunfang and Qu, Yongcun and Zhang, Liwen and Shi, Junchao and Zhang, Xudong and Liu, Shichao and Kim, Bo Hyun and Hwang, Sung Jin and Zhou, Tong and Chen, Qi and Ward, Sean M and Duan, Enkui and Zhang, Ying}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Biology of reproduction}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{99}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{6}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1266--1275}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2018}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">dec</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Oxford University Press}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1093/biolre/ioy155}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://academic.oup.com/biolreprod/article/99/6/1266/5049471}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1093/biolre/ioy155" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1093/biolre/ioy155" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr h-100"><abbr class="badge">RNA</abbr></div> <div id="zhou2018rat" class="col-sm-8 h-100"> <div class="title">Rat BodyMap transcriptomes reveal unique circular RNA features across tissue types and developmental stages</div> <div class="author"> <a href="https://tongzhoulab.org/" rel="external nofollow noopener" target="_blank">Tong Zhou</a>*#, Xueying Xie*, Musheng Li*, <b>Junchao Shi</b>, Jin J Zhou, Kenneth S Knox, Ting Wang, <a href="http://qichen-lab.info/" rel="external nofollow noopener" target="_blank">Qi Chen</a>, and Wanjun Gu#</div> <div class="periodical"> <em>Rna</em> Nov 2018 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://rnajournal.cshlp.org/content/24/11/1443" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2018/zhou2018rat.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>Circular RNAs (circRNAs) are a novel class of regulatory RNAs. Here, we present a comprehensive investigation of circRNA expression profiles across 11 tissues and four developmental stages in rats, along with cross-species analyses in humans and mice. Although the expression of circRNAs is positively correlated with that of cognate mRNAs, highly expressed genes tend to splice a larger fraction of circular transcripts. Moreover, circRNAs exhibit higher tissue specificity than cognate mRNAs. Intriguingly, while we observed a monotonic increase of circRNA abundance with age in the rat brain, we further discovered a dynamic, age-dependent pattern of circRNA expression in the testes that is characterized by a dramatic increase with advancing stages of sexual maturity and a decrease with aging. The age-sensitive testicular circRNAs are highly associated with spermatogenesis, independent of cognate mRNA expression. The tissue/age implications of circRNAs suggest that they present unique physiological functions rather than simply occurring as occasional by-products of gene transcription.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">zhou2018rat</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Rat BodyMap transcriptomes reveal unique circular RNA features across tissue types and developmental stages}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Zhou, Tong and Xie, Xueying and Li, Musheng and Shi, Junchao and Zhou, Jin J and Knox, Kenneth S and Wang, Ting and Chen, Qi and Gu, Wanjun}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Rna}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{24}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{11}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1443--1456}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2018}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">nov</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Cold Spring Harbor Lab}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1261/rna.067132.118}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://rnajournal.cshlp.org/content/24/11/1443}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1261/rna.067132.118" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1261/rna.067132.118" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr h-100"><abbr class="badge">Cell</abbr></div> <div id="wang2018asymmetric" class="col-sm-8 h-100"> <div class="title">Asymmetric expression of LincGET biases cell fate in two-cell mouse embryos</div> <div class="author">Jiaqiang Wang*, Leyun Wang*, Guihai Feng*, Yukai Wang*, Yufei Li, Xin Li, Chao Liu, Guanyi Jiao, Cheng Huang, <b>Junchao Shi</b>, <a href="https://tongzhoulab.org/" rel="external nofollow noopener" target="_blank">Tong Zhou</a>, <a href="http://qichen-lab.info/" rel="external nofollow noopener" target="_blank">Qi Chen</a>, Zhonghua Liu, Wei Li#, and Qi Zhou#</div> <div class="periodical"> <em>Cell</em> Dec 2018 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.sciencedirect.com/science/article/pii/S0092867418315642" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2018/zhou2018rat.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>In early mammalian embryos, it remains unclear how the first cell fate bias is initially triggered and amplified toward cell fate segregation. Here, we report that a long noncoding RNA, LincGET, is transiently and asymmetrically expressed in the nucleus of two- to four-cell mouse embryos. Overexpression of LincGET in one of the two-cell blastomeres biases its progeny predominantly toward the inner cell mass (ICM) fate. Mechanistically, LincGET physically binds to CARM1 and promotes the nuclear localization of CARM1, which can further increase the level of H3 methylation at Arginine 26 (H3R26me), activate ICM-specific gene expression, upregulate transposons, and increase global chromatin accessibility. Simultaneous overexpression of LincGET and depletion of Carm1 no longer biased embryonic fate, indicating that the effect of LincGET in directing ICM lineage depends on CARM1. Thus, our data identify LincGET as one of the earliest known lineage regulators to bias cell fate in mammalian 2-cell embryos.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">wang2018asymmetric</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Asymmetric expression of LincGET biases cell fate in two-cell mouse embryos}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Wang, Jiaqiang and Wang, Leyun and Feng, Guihai and Wang, Yukai and Li, Yufei and Li, Xin and Liu, Chao and Jiao, Guanyi and Huang, Cheng and Shi, Junchao and Zhou, Tong and Chen, Qi and Liu, Zhonghua and Li, Wei and Zhou, Qi}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Cell}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{175}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{7}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1887--1901}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2018}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">dec</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Cell Press}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.cell.2018.11.039}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.sciencedirect.com/science/article/pii/S0092867418315642}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1016/j.cell.2018.11.039" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1016/j.cell.2018.11.039" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li> </ol> <h2 class="year">2017</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr h-100"><abbr class="badge">Phys Life Rev</abbr></div> <div id="shi2017epigenetic" class="col-sm-8 h-100"> <div class="title">Epigenetic information in gametes: Gaming from before fertilization</div> <div class="author"> <b>Junchao Shi</b>, Xudong Zhang, Ying Liu, and <a href="http://qichen-lab.info/" rel="external nofollow noopener" target="_blank">Qi Chen</a># </div> <div class="periodical"> <em>Physics of Life Reviews</em> Mar 2017 </div> <div class="links"> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.sciencedirect.com/science/article/pii/S1571064517300015" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2017/shi2017epigenetic.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">shi2017epigenetic</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Epigenetic information in gametes: Gaming from before fertilization}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Shi, Junchao and Zhang, Xudong and Liu, Ying and Chen, Qi}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Physics of Life Reviews}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{20}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{146--149}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2017}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">mar</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Elsevier}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.plrev.2017.01.001}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.sciencedirect.com/science/article/pii/S1571064517300015}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1016/j.plrev.2017.01.001" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1016/j.plrev.2017.01.001" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr h-100"><abbr class="badge">Cell Res</abbr></div> <div id="zhang2017tsrnas" class="col-sm-8 h-100"> <div class="title">tsRNAs: new players in mammalian retrotransposon control</div> <div class="author">Yunfang Zhang, <b>Junchao Shi</b>, and <a href="http://qichen-lab.info/" rel="external nofollow noopener" target="_blank">Qi Chen</a># </div> <div class="periodical"> <em>Cell research</em> Nov 2017 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.nature.com/articles/cr2017109" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2017/zhang2017tsrnas.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>A recent study led by Professor Rob Martienssen in Cell showed that 3’-tRNA-derived small RNAs can suppress long terminal repeat retrotransposon activity in mammalian cells by mechanisms independent of DNA-associated epigenetic marks, suggesting how the genome may defend itself from retrotransposon invasion during epigenetic reprogramming.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">zhang2017tsrnas</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{tsRNAs: new players in mammalian retrotransposon control}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Zhang, Yunfang and Shi, Junchao and Chen, Qi}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Cell research}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{27}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{11}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1307--1308}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2017}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">nov</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Nature Publishing Group}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/cr.2017.109}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.nature.com/articles/cr2017109}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1038/cr.2017.109" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1038/cr.2017.109" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li> </ol> <a name="works_at_IOZ"></a> <h1>Works at UCAS</h1> <h2 class="year">2017</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr h-100"><abbr class="badge">Nat Commun</abbr></div> <div id="liu2017bcas2" class="col-sm-8 h-100"> <div class="title">BCAS2 is involved in alternative mRNA splicing in spermatogonia and the transition to meiosis</div> <div class="author">Wenbo Liu, Fengchao Wang, Qianhua Xu, <b>Junchao Shi</b>, Xiaoxin Zhang, Xukun Lu, Zhen-Ao Zhao, Zheng Gao, Huaixiao Ma, Enkui Duan, Fei Gao, Shaorong Gao#, Zhaohong Yi#, and Lei Li#</div> <div class="periodical"> <em>Nature communications</em> Jan 2017 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.nature.com/articles/ncomms14182" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2017/liu2017bcas2.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>Breast cancer amplified sequence 2 (BCAS2) is involved in multiple biological processes, including pre-mRNA splicing. However, the physiological roles of BCAS2 are still largely unclear. Here we report that BCAS2 is specifically enriched in spermatogonia of mouse testes. Conditional disruption of Bcas2 in male germ cells impairs spermatogenesis and leads to male mouse infertility. Although the spermatogonia appear grossly normal, spermatocytes in meiosis prophase I and meiosis events (recombination and synapsis) are rarely observed in the BCAS2-depleted testis. In BCAS2 null testis, 245 genes are altered in alternative splicing forms; at least three spermatogenesis-related genes (Dazl, Ehmt2 and Hmga1) can be verified. In addition, disruption of Bcas2 results in a significant decrease of the full-length form and an increase of the short form (lacking exon 8) of DAZL protein. Altogether, our results suggest that BCAS2 regulates alternative splicing in spermatogonia and the transition to meiosis initiation, and male fertility.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">liu2017bcas2</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{BCAS2 is involved in alternative mRNA splicing in spermatogonia and the transition to meiosis}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Liu, Wenbo and Wang, Fengchao and Xu, Qianhua and Shi, Junchao and Zhang, Xiaoxin and Lu, Xukun and Zhao, Zhen-Ao and Gao, Zheng and Ma, Huaixiao and Duan, Enkui and Gao, Fei and Gao, Shaorong and Yi, Zhaohong and Li, Lei}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature communications}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{8}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1--11}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2017}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jan</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Nature Publishing Group}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/ncomms14182}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.nature.com/articles/ncomms14182}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1038/ncomms14182" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1038/ncomms14182" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr h-100"> <abbr class="badge">Sci China Life Sci</abbr><abbr2 class="award badge">Chinese</abbr2> </div> <div id="shi2017mammalian" class="col-sm-8 h-100"> <div class="title">Mammalian transgenerational inheritance of acquired traits via germline transmission</div> <div class="author"> <b>Junchao Shi</b>, <a href="http://qichen-lab.info/" rel="external nofollow noopener" target="_blank">Qi Chen</a>, and Enkui Duan#</div> <div class="periodical"> <em>Chinese Bulletin of Life Sciences</em> Jan 2017 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.cnki.net/kcms/doi/10.13376/j.cbls/2017002.html" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2017/shi2017mammalian.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>Increasing evidence indicates that certain parental environmental exposures, such as chemical contact, diet change and mental stress can be "memorized" and passed to future generations, suggesting epigenetic inheritance via germline. Such phenomenon raised significant interests and led to a resurrected interests to the once heretical concept of "Lamarckism". The potential "epigenetic carriers" in the germline include DNA methylation, chromatin structure/histone modifications and noncoding RNAs, further elucidating the molecular mechanism of transgenerational inheritance via mammalian germline would have a profound impact on our understanding of many modern disease etiology.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">shi2017mammalian</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Mammalian transgenerational inheritance of acquired traits via germline transmission}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Shi, Junchao and Chen, Qi and Duan, Enkui}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Chinese Bulletin of Life Sciences}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2017}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jan</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.13376/j.cbls/2017002}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.13376/j.cbls/2017002" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.13376/j.cbls/2017002" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li> </ol> <h2 class="year">2016</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr h-100"><abbr class="badge">Science</abbr></div> <div id="chen2016sperm" class="col-sm-8 h-100"> <div class="title">Sperm tsRNAs contribute to intergenerational inheritance of an acquired metabolic disorder</div> <div class="author"> <a href="http://qichen-lab.info/" rel="external nofollow noopener" target="_blank">Qi Chen</a>*, Menghong Yan*, Zhonghong Cao*, Xin Li*, Yunfang Zhang*, <b>Junchao Shi</b>*, Gui-hai Feng, Hongying Peng, Xudong Zhang, Ying Zhang, Jingjing Qian, Enkui Duan#, Qiwei Zhai#, and Qi Zhou#</div> <div class="periodical"> <em>Science</em> Jan 2016 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.science.org/doi/10.1126/science.aad7977" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2016/chen2016sperm.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>Increasing evidence indicates that metabolic disorders in offspring can result from the father’s diet, but the mechanism remains unclear. In a paternal mouse model given a high-fat diet (HFD), we showed that a subset of sperm transfer RNA-derived small RNAs (tsRNAs), mainly from 5’ transfer RNA halves and ranging in size from 30 to 34 nucleotides, exhibited changes in expression profiles and RNA modifications. Injection of sperm tsRNA fractions from HFD males into normal zygotes generated metabolic disorders in the F1 offspring and altered gene expression of metabolic pathways in early embryos and islets of F1 offspring, which was unrelated to DNA methylation at CpG-enriched regions. Hence, sperm tsRNAs represent a paternal epigenetic factor that may mediate intergenerational inheritance of diet-induced metabolic disorders.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">chen2016sperm</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Sperm tsRNAs contribute to intergenerational inheritance of an acquired metabolic disorder}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Chen, Qi and Yan, Menghong and Cao, Zhonghong and Li, Xin and Zhang, Yunfang and Shi, Junchao and Feng, Gui-hai and Peng, Hongying and Zhang, Xudong and Zhang, Ying and Qian, Jingjing and Duan, Enkui and Zhai, Qiwei and Zhou, Qi}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Science}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{351}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{6271}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{397--400}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2016}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jan</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{American Association for the Advancement of Science}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1126/science.aad7977}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.science.org/doi/10.1126/science.aad7977}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1126/science.aad7977" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1126/science.aad7977" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr h-100"><abbr class="badge">J Mol Cell Biol</abbr></div> <div id="liu2016btg4" class="col-sm-8 h-100"> <div class="title">BTG4 is a key regulator for maternal mRNA clearance during mouse early embryogenesis</div> <div class="author">Yusheng Liu*, Xukun Lu*, <b>Junchao Shi</b>, Xingjiang Yu, Xiaoxin Zhang, Kai Zhu, Zhaohong Yi, Enkui Duan, and Lei Li#</div> <div class="periodical"> <em>Journal of molecular cell biology</em> Aug 2016 </div> <div class="links"> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://academic.oup.com/jmcb/article/8/4/366/2588585" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2016/liu2016btg4.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">liu2016btg4</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{BTG4 is a key regulator for maternal mRNA clearance during mouse early embryogenesis}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Liu, Yusheng and Lu, Xukun and Shi, Junchao and Yu, Xingjiang and Zhang, Xiaoxin and Zhu, Kai and Yi, Zhaohong and Duan, Enkui and Li, Lei}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Journal of molecular cell biology}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{8}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{4}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{366--368}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2016}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">aug</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Chinese Academy of Sciences}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1093/jmcb/mjw023}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://academic.oup.com/jmcb/article/8/4/366/2588585}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1093/jmcb/mjw023" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1093/jmcb/mjw023" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr h-100"><abbr class="badge">Environ Epigenet</abbr></div> <div id="shi2016molecular" class="col-sm-8 h-100"> <div class="title">Molecular carriers of acquired inheritance: absence of evidence is not evidence of absence</div> <div class="author"> <b>Junchao Shi</b>*, Yunfang Zhang*, and <a href="http://qichen-lab.info/" rel="external nofollow noopener" target="_blank">Qi Chen</a># </div> <div class="periodical"> <em>Environmental Epigenetics</em> Apr 2016 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://academic.oup.com/eep/article/2/2/dvw014/2841062" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2016/shi2016molecular.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>In utero exposure to environmental endocrine disruptors can cause transgenerational effects in the males of subsequent generations. DNA methylation (5 mC) was suggested, but being challenged as the molecular carrier of such epigenetic information. In a recent study, Schuster et al. show a changed small RNA profile changed in the sperm of F3 generation after F0 in utero vinclozolin exposure, suggesting additional transgenerational epigenetic carriers for endocrine disruptor effects, other than DNA methylation.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">shi2016molecular</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Molecular carriers of acquired inheritance: absence of evidence is not evidence of absence}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Shi, Junchao and Zhang, Yunfang and Chen, Qi}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Environmental Epigenetics}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{2}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{2}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{dvw014}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2016}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">apr</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Oxford University Press}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1093/eep/dvw014}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://academic.oup.com/eep/article/2/2/dvw014/2841062}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1093/eep/dvw014" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1093/eep/dvw014" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li> </ol> <h2 class="year">2015</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr h-100"><abbr class="badge">Development</abbr></div> <div id="shi2015dynamic" class="col-sm-8 h-100"> <div class="title">Dynamic transcriptional symmetry-breaking in pre-implantation mammalian embryo development revealed by single-cell RNA-seq</div> <div class="author"> <b>Junchao Shi</b>*, <a href="http://qichen-lab.info/" rel="external nofollow noopener" target="_blank">Qi Chen</a>*#, Xin Li*, Xiudeng Zheng*, Ying Zhang, Jie Qiao, Fuchou Tang, Yi Tao#, Qi Zhou#, and Enkui Duan#</div> <div class="periodical"> <em>Development</em> Oct 2015 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://journals.biologists.com/dev/article/142/20/3468/47006/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2015/shi2015dynamic.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>During mammalian pre-implantation embryo development, when the first asymmetry emerges and how it develops to direct distinct cell fates remain longstanding questions. Here, by analyzing single-blastomere transcriptome data from mouse and human pre-implantation embryos, we revealed that the initial blastomere-to-blastomere biases emerge as early as the first embryonic cleavage division, following a binomial distribution pattern. The subsequent zygotic transcriptional activation further elevated overall blastomere-to-blastomere biases during the two- to 16-cell embryo stages. The trends of transcriptional asymmetry fell into two distinct patterns: for some genes, the extent of asymmetry was minimized between blastomeres (monostable pattern), whereas other genes, including those known to be lineage specifiers, showed ever-increasing asymmetry between blastomeres (bistable pattern), supposedly controlled by negative or positive feedbacks. Moreover, our analysis supports a scenario in which opposing lineage specifiers within an early blastomere constantly compete with each other based on their relative ratio, forming an inclined ’lineage strength’ that pushes the blastomere onto a predisposed, yet flexible, lineage track before morphological distinction.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">shi2015dynamic</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Dynamic transcriptional symmetry-breaking in pre-implantation mammalian embryo development revealed by single-cell RNA-seq}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Shi, Junchao and Chen, Qi and Li, Xin and Zheng, Xiudeng and Zhang, Ying and Qiao, Jie and Tang, Fuchou and Tao, Yi and Zhou, Qi and Duan, Enkui}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Development}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{142}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{20}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{3468--3477}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2015}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">oct</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{The Company of Biologists}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1242/dev.123950}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://journals.biologists.com/dev/article/142/20/3468/47006/}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1242/dev.123950" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1242/dev.123950" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr h-100"><abbr class="badge">Cell Mol Life Sci</abbr></div> <div id="liu2015senescence" class="col-sm-8 h-100"> <div class="title">Senescence of human skin-derived precursors regulated by Akt-FOXO3-p27KIP1/p15INK4b signaling</div> <div class="author">Shuang Liu*, Xinyue Wang*, Qian Zhao*, Shu Liu, Huishan Zhang, <b>Junchao Shi</b>, Na Li, Xiaohua Lei, Huashan Zhao, Zhili Deng, Yujing Cao, Lina Ning, Guoliang Xia#, and Enkui Duan#</div> <div class="periodical"> <em>Cellular and Molecular Life Sciences</em> Aug 2015 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://link.springer.com/article/10.1007/s00018-015-1877-3" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2015/liu2015senescence.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>Multipotent skin-derived precursors (SKPs) are dermal stem cells with the capacity to reconstitute the dermis and other tissues, such as muscles and the nervous system. Thus, the easily available human SKPs (hSKPs) hold great promises in regenerative medicine. However, long-term expansion is difficult for hSKPs in vitro. We previously demonstrated that hSKPs senesced quickly under routine culture conditions. To identify the underlying mechanisms so as to find an effective way to expand hSKPs, time-dependent microarray analysis of gene expression in hSKPs during in vitro culture was performed. We found that the senescence of hSKPs had a unique gene expression pattern that differs from reported typical senescence. Subsequent investigation ruled out the role of DNA damage and classical p53 and p16(INK4a) signaling in hSKP senescence. Examination of cyclin-dependent kinase inhibitors revealed the involvement of p15(INK4b) and p27(KIP1). Further exploration about upstream signals indicated the contribution of Akt hypo-activity and FOXO3 to hSKP senescence. Forced activation of Akt and knockdown of FOXO3, p15(INK4b) and p27(KIP1) effectively inhibited hSKP senescence and promoted hSKP proliferation. The unique senescent phenotype of human dermal stem cells and the role of Akt-FOXO3-p27(KIP1)/p15(INK4b) signaling in regulating hSKP senescence provide novel insights into the senescence and self-renewal regulation of adult stem cells. The present study also points out a way to propagate hSKPs in vitro so as to fulfill their promises in regenerative medicine.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">liu2015senescence</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Senescence of human skin-derived precursors regulated by Akt-FOXO3-p27KIP1/p15INK4b signaling}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Liu, Shuang and Wang, Xinyue and Zhao, Qian and Liu, Shu and Zhang, Huishan and Shi, Junchao and Li, Na and Lei, Xiaohua and Zhao, Huashan and Deng, Zhili and Cao, Yujing and Ning, Lina and Xia, Guoliang and Duan, Enkui}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Cellular and Molecular Life Sciences}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{72}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{15}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{2949--2960}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2015}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">aug</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Springer Basel}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/s00018-015-1877-3}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://link.springer.com/article/10.1007/s00018-015-1877-3}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1007/s00018-015-1877-3" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1007/s00018-015-1877-3" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li> </ol> <h2 class="year">2014</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr h-100"> <img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/2014_JMCB_cover.gif"> <div class="w-100"></div> <abbr2 class="award badge">Cover Story</abbr2> </div> <div id="zhang2014identification" class="col-sm-8 h-100"> <div class="title">Identification and characterization of an ancient class of small RNAs enriched in serum associating with active infection</div> <div class="author">Yunfang Zhang*, Ying Zhang*, <b>Junchao Shi</b>*, He Zhang, Zhonghong Cao, Xuan Gao, Wanhua Ren, Yunna Ning, Lina Ning, Yujing Cao, Yongchang Chen, Weizhi Ji, Zi-jiang Chen#, <a href="http://qichen-lab.info/" rel="external nofollow noopener" target="_blank">Qi Chen</a>#, and Enkui Duan#</div> <div class="periodical"> <em>Journal of molecular cell biology</em> Apr 2014 </div> <div class="links"> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://academic.oup.com/jmcb/article/6/2/172/964660" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2014/zhang2014identification.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">zhang2014identification</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Identification and characterization of an ancient class of small RNAs enriched in serum associating with active infection}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Zhang, Yunfang and Zhang, Ying and Shi, Junchao and Zhang, He and Cao, Zhonghong and Gao, Xuan and Ren, Wanhua and Ning, Yunna and Ning, Lina and Cao, Yujing and Chen, Yongchang and Ji, Weizhi and Chen, Zi-jiang and Chen, Qi and Duan, Enkui}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Journal of molecular cell biology}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{6}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{2}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{172--174}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2014}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">apr</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Oxford University Press}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1093/jmcb/mjt052}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://academic.oup.com/jmcb/article/6/2/172/964660}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1093/jmcb/mjt052" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1093/jmcb/mjt052" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr h-100"> <img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/2014_CR_cover.jpg"> <div class="w-100"></div> <abbr2 class="award badge">Cover Story</abbr2> </div> <div id="zhang2014uterine" class="col-sm-8 h-100"> <div class="title">Uterine Rbpj is required for embryonic-uterine orientation and decidual remodeling via Notch pathway-independent and-dependent mechanisms</div> <div class="author">Shuang Zhang*, Shuangbo Kong*, Bingyan Wang, Xiaohong Cheng, Yongjie Chen, Weiwei Wu, Qiang Wang, <b>Junchao Shi</b>, Ying Zhang, Shumin Wang, Jinhua Lu, John P Lydon, Francesco DeMayo, Warren S Pear, Hua Han, Haiyan Lin, Hongmei Li, Yan-ling Wang, Bing Li, <a href="http://qichen-lab.info/" rel="external nofollow noopener" target="_blank">Qi Chen</a>, Enkui Duan#, and Haibin Wang#</div> <div class="periodical"> <em>Cell research</em> Aug 2014 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.nature.com/articles/cr201482" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2014/zhang2014uterine.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>Coordinated uterine-embryonic axis formation and decidual remodeling are hallmarks of mammalian post-implantation embryo development. Embryonic-uterine orientation is determined at initial implantation and synchronized with decidual development. However, the molecular mechanisms controlling these events remain elusive despite its discovery a long time ago. In the present study, we found that uterine-specific deletion of Rbpj, the nuclear transducer of Notch signaling, resulted in abnormal embryonic-uterine orientation and decidual patterning at post-implantation stages, leading to substantial embryo loss. We further revealed that prior to embryo attachment, Rbpj confers on-time uterine lumen shape transformation via physically interacting with uterine estrogen receptor (ERα) in a Notch pathway-independent manner, which is essential for the initial establishment of embryo orientation in alignment with uterine axis. While at post-implantation stages, Rbpj directly regulates the expression of uterine matrix metalloproteinase in a Notch pathway-dependent manner, which is required for normal post-implantation decidual remodeling. These results demonstrate that uterine Rbpj is essential for normal embryo development via instructing the initial embryonic-uterine orientation and ensuring normal decidual patterning in a stage-specific manner. Our data also substantiate the concept that normal mammalian embryonic-uterine orientation requires proper guidance from developmentally controlled uterine signaling.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">zhang2014uterine</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Uterine Rbpj is required for embryonic-uterine orientation and decidual remodeling via Notch pathway-independent and-dependent mechanisms}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Zhang, Shuang and Kong, Shuangbo and Wang, Bingyan and Cheng, Xiaohong and Chen, Yongjie and Wu, Weiwei and Wang, Qiang and Shi, Junchao and Zhang, Ying and Wang, Shumin and Lu, Jinhua and Lydon, John P and DeMayo, Francesco and Pear, Warren S and Han, Hua and Lin, Haiyan and Li, Lei an Wang, Hongmei and Wang, Yan-ling and Li, Bing and Chen, Qi and Duan, Enkui and Wang, Haibin}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Cell research}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{24}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{8}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{925--942}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2014}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">aug</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Nature Publishing Group}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/cr.2014.82}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.nature.com/articles/cr201482}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1038/cr.2014.82" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1038/cr.2014.82" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li> </ol> <h2 class="year">2013</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr h-100"><abbr class="badge">PLoS One</abbr></div> <div id="zhang2013hormonal" class="col-sm-8 h-100"> <div class="title">Hormonal regulation of ovarian bursa fluid in mice and involvement of aquaporins</div> <div class="author">He Zhang*, Ying Zhang*, Huashan Zhao, Yunfang Zhang, <a href="http://qichen-lab.info/" rel="external nofollow noopener" target="_blank">Qi Chen</a>, Hongying Peng, Li Lei, Jingqiao Qiao, <b>Junchao Shi</b>, Zhonghong Cao, Enkui Duan#, and Yaping Jin#</div> <div class="periodical"> <em>PLoS One</em> Aug 2013 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0063823" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2013/zhang2013hormonal.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>In rodent species, the ovary and the end of oviduct are encapsulated by a thin membrane called ovarian bursa. The biological functions of ovarian bursa remain unexplored despite its structural arrangement in facilitating oocytes transport into oviduct. In the present study, we observed a rapid fluid accumulation and reabsorption within the ovarian bursa after ovarian stimulation (PMSG-primed hCG injection), suggesting that the ovarian bursa might play an active role in regulating local fluid homeostasis around the timing of ovulation. We hypothesized that the aquaporin proteins, which are specialized channels for water transport, might be involved in this process. By screening the expression of aquaporin family members (Aqp1-9) in the ovarian tissue and isolated ovarian bursa (0, 1, 2 and 5 h after hCG injection), we found that AQP2 and AQP5 mRNA showed dynamic changes after hCG treatment, showing upregulation at 1-2 h followed by gradually decrease at 5 h, which is closely related with the intra-bursa fluid dynamics. Further immunofluorescence examinations of AQP2 and AQP5 in the ovarian bursa revealed that AQP2 is specifically localized in the outer layer (peritoneal side) while AQP5 localized in the inner layer (ovarian side) of the bursa, such cell type specific and spatial-temporal expressions of AQP2 and 5 support our hypothesis that they might be involved in efficient water transport through ovarian bursa under ovulation related hormonal regulation. The physiological significance of aquaporin-mediated water transport in the context of ovarian bursa still awaits further clarification.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">zhang2013hormonal</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Hormonal regulation of ovarian bursa fluid in mice and involvement of aquaporins}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Zhang, He and Zhang, Ying and Zhao, Huashan and Zhang, Yunfang and Chen, Qi and Peng, Hongying and Lei, Li and Qiao, Jingqiao and Shi, Junchao and Cao, Zhonghong and Duan, Enkui and Jin, Yaping}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{PLoS One}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{8}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{5}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{e63823}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2013}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Public Library of Science San Francisco, USA}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1371/journal.pone.0063823}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0063823}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1371/journal.pone.0063823" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1371/journal.pone.0063823" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li></ol> <h2 class="year">2012</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr h-100"><abbr class="badge">Cell Res</abbr></div> <div id="peng2012novel" class="col-sm-8 h-100"> <div class="title">A novel class of tRNA-derived small RNAs extremely enriched in mature mouse sperm</div> <div class="author">Hongying Peng*, <b>Junchao Shi</b>*, Ying Zhang, He Zhang, Shangying Liao, Wei Li, Li Lei, Chunsheng Han, Lina Ning, Yujing Cao, Qi Zhou, <a href="http://qichen-lab.info/" rel="external nofollow noopener" target="_blank">Qi Chen</a>#, and Enkui Duan#</div> <div class="periodical"> <em>Cell research</em> Nov 2012 </div> <div class="links"> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.nature.com/articles/cr2012141" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf//2012/peng2012novel.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">peng2012novel</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{A novel class of tRNA-derived small RNAs extremely enriched in mature mouse sperm}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Peng, Hongying and Shi, Junchao and Zhang, Ying and Zhang, He and Liao, Shangying and Li, Wei and Lei, Li and Han, Chunsheng and Ning, Lina and Cao, Yujing and Zhou, Qi and Chen, Qi and Duan, Enkui}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Cell research}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{22}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{11}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1609--1612}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2012}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">nov</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Nature Publishing Group}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/cr.2012.141}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.nature.com/articles/cr2012141}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> <div class="altmetric col-sm-2 h-100"> <div style="text-align: right;" data-badge-popover="left" data-badge-type="donut" data-doi="10.1038/cr.2012.141" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed"></div> <span style="text-align: right;" class="__dimensions_badge_embed__" data-doi="10.1038/cr.2012.141" data-hide-zero-citations="true" data-style="small_rectangle"></span> </div> </div> </li></ol> </div> </article> </div> </div> <footer class="sticky-bottom mt-5"> <div class="container"> © Copyright 2025 Shi Lab. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with modified <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Last updated: March 18, 2025. </div> </footer> <div id="backtop"> <a href="#">⤒</a> </div> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js" integrity="sha256-fgLAgv7fyCGopR/gBNq2iW3ZKIdqIcyshnUULC4vex8=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js"></script> <script defer src="/assets/js/common.js"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script async src="https://www.googletagmanager.com/gtag/js?id=G-FR0W4CMM4L"></script> <script>function gtag(){window.dataLayer.push(arguments)}window.dataLayer=window.dataLayer||[],gtag("js",new Date),gtag("config","G-FR0W4CMM4L");</script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>